soh/src/overlays/actors/ovl_En_Weiyer/z_en_weiyer.c

```diff
@@ -624,6 +624,8 @@ void EnWeiyer_Update(Actor* thisx, PlayState* play) {
 
 s32 EnWeiyer_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, void* thisx,
                               Gfx** gfx) {
+    return 0;
```

Comment:
Is this return necessary? The non-water based stinger (`EnEiyer`) has a similar `pos->z` addition that you didn't remove and both actors use the same DLs from what I see. Was this just left over testing that is not needed?